aries-site/src/pages/foundation/tshirt-sizing.mdx

```diff
@@ -47,17 +51,74 @@ xxlarge: `${baseSpacing * 64}px`, // 1536px
+- **Scale consists of a primary scale** based on the doubling of a 24px base unit (same as v6) + a supplementary scale doubling a 16px base unit
+
+**Results in the following pixel progression:**
+1, 2, 3, 4, 6, 8, 12, 16, 24, 32, 48, 64, 96, 128, 192, 256, 384, 512, 768, 1024, 1536, 2048
+
+Below are the sizing values from the [HPE theme v7](https://github.com/grommet/grommet-theme-hpe/blob/master/CHANGELOG.md#700) and above which implements this expanded scale:
+
+### Container sizes comparison (width, height, size, columns, rows)
+
+| **v6 and below** | **v7**    | **Value** | **Notes**                                 |
+| ---------------- | --------- | --------- | ----------------------------------------- |
+| `xxsmall`        | `5xsmall` | 48px      | Renamed for consistency                   |
+| --               | `4xsmall` | 64px      | New intermediate size                     |
+| `xsmall`         | `3xsmall` | 96px      | Renamed for consistency                   |
+| --               | `xxsmall` | 128px     | New intermediate size                     |
+| `small`          | `xsmall`  | 192px     | Renamed for consistency                   |
+| --               | `small`   | 256px     | New intermediate size                     |
+| `medium`         | `medium`  | 384px     | **Unchanged**                             |
+| --               | `large`   | 512px     | New intermediate size                     |
+| `large`          | `xlarge`  | 768px     | Renamed for consistency                   |
+| --               | `xxlarge` | 1024px    | New intermediate size                     |
+| `xlarge`         | --        | 1152px    | Value eliminated (doesn't follow formula) |
+| `xxlarge`        | `3xlarge` | 1536px    | Renamed for consistency                   |
+
+### Spacing sizes comparison (pad, margin, gap, thickness)
+
+| **v6 and below** | **v7**    | **Value** | **Notes**               |
+| ---------------- | --------- | --------- | ----------------------- |
+| `hair`           | `hair`    | 1px       | Unchanged               |
+| `xxsmall`        | `5xsmall` | 3px       | Renamed for consistency |
+| --               | `4xsmall` | 4px       | New intermediate size   |
+| `xsmall`         | `3xsmall` | 6px       | Renamed for consistency |
+| --               | `xxsmall` | 8px       | New intermediate size   |
+| `small`          | `xsmall`  | 12px      | Renamed for consistency |
+| --               | `small`   | 16px      | New intermediate size   |
+| `medium`         | `medium`  | 24px      | **Unchanged**           |
+| --               | `large`   | 32px      | New intermediate size   |
+| `large`          | `xlarge`  | 48px      | Renamed for consistency |
+| --               | `xxlarge` | 64px      | New intermediate size   |
+| `xlarge`         | `3xlarge` | 96px      | Renamed for consistency |
+
 ## T-shirt sizing for component dimensions
```

Comment:
One we reach this section it's not clear if we are referencing values from the original scale or the dual doubling scale. I think the page structure and hierarchy should be revisited to make this more clear.